openwisp_ipam/migrations/0005_default_groups_permissions.py

```diff
@@ -1,16 +1,23 @@
+from django.conf import settings
 from django.db import migrations
+from openwisp_users.migrations import (
+    create_default_groups as base_create_default_groups,
+)
 
 from openwisp_ipam.migrations import assign_permissions_to_groups
 
 
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('openwisp_users', '0004_default_groups'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
         ('openwisp_ipam', '0004_subnet_organization_unique_together'),
     ]
 
     operations = [
+        migrations.RunPython(
+            base_create_default_groups, reverse_code=migrations.RunPython.noop
+        ),
```

Comment:
Ran `create_default_groups` from `openwisp_users` to ensure permissions are set correctly.